aws/templates/commonApplication.ftl

```diff
@@ -726,6 +735,59 @@
 
         [#assign _context += getFinalEnvironment(task, _context) ]
 
+        [#-- validate fargate requirements from container context --]
+        [#if solution.Engine == "fargate" ]
+            [#assign fargateInvalidConfig = false ]
+            [#assign fargateInvalidConfigMessage = [] ]
+            
+            [#if !( [ 256, 512, 1024, 2048, 4096 ]?seq_contains(_context.Cpu) )  ]
+                [#assign fargateInvalidConfig = true ]
+                [#assign fargateInvalidConfigMessage += [ "CPU quota is not valid ( must be divisible by 256 )" ] ] 
+            [/#if]
+
+            [#-- && ((( _context.MaximumMemory!1 % 256 )?string["0.#"])?split(".")[1]) != "0" --]
+            [#if !( _context.MaximumMemory?has_content )  ] 
+                [#local  memoryRemainder = ( _context.MaximumMemory!1 % 256 )?string["0.#"]]
```

Comment:
max memory not present so this is redundant - looks like an incomplete refactor? Maybe just drop 256 check and just report if max memory not provided?

Reply:
fixed